src/lib/snykGroup.ts

```diff
@@ -348,7 +410,17 @@ export class snykGroup {
               if (org.name == um.org) {
                 orgMatch = true;
                 if (org.role.toUpperCase() == um.role.toUpperCase()) {
-                  roleMatch = true;
+                  if (
```

Comment:
FYI this is the bit of logic that causes the API to be called regardless of whether the membership file role matches the user role, which is necessary since they can match (I.e admin = admin) but in reality be two different roles (org admin vs custom role admin)